### PR TITLE
Simple-payments blocks: fix tests

### PR DESCRIPTION
**Pull request by glendaviesnz**
#### Changes proposed in this Pull Request:
* Removes some test fixtures that were committed by mistake, and updates currency on unit tests as the USD currency display differently depending on which env the test is run in

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Check out PR and run `yarn test-extensions` and ensure all tests complete successfully

### CODE REVIEW

File: projects/plugins/jetpack/extensions/blocks/simple-payments/test/validate.js [hunk 1]
```diff
@@ -4,6 +4,16 @@
 import { settings } from '../';
 import runBlockFixtureTests from '../../../shared/test/block-fixtures';
 
+beforeEach(() => {
+    Intl.NumberFormat = jest
+        .fn()
+        .mockImplementation(() => ({format: value => `A$${value.toString()}.00`}));
```
Comment (ramonjd) [inline projects/plugins/jetpack/extensions/blocks/simple-payments/test/validate.js:10]: Curious that the linter doesn't complain about expected spaces between `() {}` etc

**Review by ramonjd — DISMISSED** [review]
All tests pass. Thanks for debugging the differences we were seeing between envs!

File: projects/plugins/jetpack/extensions/blocks/simple-payments/test/validate.js (hunk 1 above)
Comment (glendaviesnz) [reply projects/plugins/jetpack/extensions/blocks/simple-payments/test/validate.js]: that is strange, have fixed now.

**Review by aaronrobertshaw — DISMISSED** [review]
LGTM 👍 

All tests are green.

**Review by jeherve — APPROVED** [review]

**Comment by glendaviesnz** [comment]
rWP224095

**Pull request merged**